Cargo.toml

```diff
@@ -58,6 +58,8 @@ slab = "0.4.7"
 tempfile = "3.3.0"
 textwrap = "0.16.0"
 thiserror = "1.0.37"
+tracing = "0.1.37"
+tracing-subscriber = "0.3.16"
```

Comment:
tracing-subscriber's default-features brings in a lot, as indicated by the 107-line Cargo.lock diff. Let's `default-features = false` this and enable only the minimally needed stuff.

Reply:
Updated with std, fmt, and ansi